lib/combobox.js

```diff
@@ -34,11 +33,12 @@ module.exports = React.createClass({
      * ```
     */
     onSelect: React.PropTypes.func,
-
```

Comment:
We should respect the formatting of the original library to prevent future merge conflicts